src/platform/pdf/pdf.ts

```diff
@@ -0,0 +1,47 @@
+import { Client } from '../../abstracts/client';
+import { Tenant } from '../tenant/tenant';
+import { CreateFromHtmlArgsDto } from './models/create-from-html-args.dto';
+import { CreateFromHtmlRequestDto } from './models/create-from-html-request.dto';
+import { CreateFromHtmlResponseDto } from './models/create-from-html-response.dto';
+import { CreatePresignedPostResponseDto } from '../file/models/create-presigned-post-response.dto';
+
+export class PdfServiceClient extends Client {
+
+  readonly tenantId: string;
+
+  private readonly BASE_URLS = {
+    'PROD': 'https://pdf-service-api.nebulr-core.com',
+    'STAGE': 'https://pdf-service-api-stage.nebulr-core.com',
+    'DEV': 'http://pdf-service-api:3000'
+  };
+
+  constructor(parentEntity: Tenant, debug = false) {
+    super(parentEntity, debug);
+    this.tenantId = parentEntity.id;
+  }
+
```

Comment:
Would be good with `createFromUrl` so a caller just needs to send an URL to the service which will crawl the url and generate PDF (Not in the scope of this ticket). What do you think?

Reply:
Already did in push